lib/active_merchant/billing/gateways/ebanx.rb

```diff
@@ -278,7 +270,7 @@ def message_from(response)
 
       def authorization_from(action, parameters, response)
         if action == :store
-          "#{response.try(:[], 'token')}|#{CARD_BRAND[parameters[:payment_type_code].to_sym]}"
+          response.try(:[], 'token')
```

Comment:
I think we should probably keep this just in case we need to change this again, otherwise we lose data for any cards stored in the meantime, and the extra info is not causing any problem

Reply:
Only on successes is fine, that makes sense because authorization strings from failed store txns wouldn't get used on followups.